core/scss/utilities/mixins/typography/_headings.scss

```diff
@@ -26,6 +25,10 @@
     font-family: $font-slab;
   }
 
+  .su-script & {
```

Comment:
Is this missing something after the `&`?

Reply:
Nope. It's in place so that it wraps the declaration before it.